src/app/copilot/components/copilot-panel/copilot-panel.component.scss

```
@@ -364,7 +364,11 @@ $ease: cubic-bezier(0.4, 0, 0.2, 1);
       padding: ($u * 2) ($u * 2 + 4px);
       font-size: 14.5px;
       line-height: 1.7;
-      overflow-wrap: break-word;
+      // 'anywhere' (unlike 'break-word') also shrinks min-content width, so an
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)